src/components/Result/ResultHeader.vue

```diff
@@ -14,9 +14,14 @@
 				>
 					{{ title }}
 				</p>
-				<p class="hidden md:flex text-xl">
+				<p v-if="description !== ''" class="hidden md:flex text-xl">
```

Comment:
maybe `v-if="description"` is sufficient and easier to read. Or `!!description` if we want to be type strict.